extensions/reviewed/DraggablePhysics.json

```
@@ -450,22 +420,338 @@
           ],
           "parameters": [
             {
-              "codeOnly": false,
-              "defaultValue": "",
               "description": "Object",
-              "longDescription": "",
               "name": "Object",
-              "optional": false,
-              "supplementaryInformation": "",
               "type": "object"
             },
             {
-              "codeOnly": false,
-              "defaultValue": "",
               "description": "Behavior",
-              "longDescription": "",
               "name": "Behavior",
-              "optional": false,
+              "supplementaryInformation": "DraggablePhysics::DraggablePhysics",
+              "type": "behavior"
+            }
+          ],
+          "objectGroups": []
+        },
+        {
+          "description": "the mouse button used to move the object.",
+          "fullName": "Mouse button",
+          "functionType": "ExpressionAndCondition",
+          "group": "Draggable (for physics objects) configuration",
+          "name": "MouseButton",
+          "sentence": "the mouse button used to move the object",
```

Because it's suffixed by "of Object".
```suggestion
          "sentence": "the dragging mouse button",
```